blog/Introducing-Femto.md

```diff
@@ -0,0 +1,184 @@
+
+### Fable Packages
+As you all know, Fable packages are normal dotnet packages that are distributed to the nuget package registry. A special type of these nuget packages are Fable bindings which provide idiomatic and type-safe wrappers around native Javascript APIs. These native Javascript APIs can be either available directly in the environment where your compiled F# code is running (Web APIs in browsers, system modules in node.js), or these APIs can be those of third-party javascript libraries distributed to the npm registry and this is where things get complicated.
+
+In order to use a Fable binding for a third-party javascript library you as a user have to install *both* libraries in your project:
+ - The Fable binding from nuget
+ - The actual javascript third-party library from npm
+
+For example when you install `Fable.React` in your Fable project you also have to install `react` and `react-dom` into your npm dependencies. Users usually have to look up the the exact versions of the npm package they need from the documentation page of the Fable binding itself, in case of `Fable.React` you need to install `react@16.8.0` and `react-dom@16.8.0`.
+
+This is of course doable if you only have a couple of packages with direct npm dependencies such as `Fable.React` but the problem gets even more complicated when you install a nuget package that itself *depends* on a Fable binding such as `Fable.Elmish.React` which depends both on `Fable.Elmish` and `Fable.React`:
+```
+Fable.Elmish.React (nuget)
+  |
+  | -- Fable.Elmish (nuget)
+  | -- Fable.React (nuget)
+        |
+        | -- react@16.8.0 (npm)
+        | -- react-dom@16.8.0 (npm)
+```
+Here, when you install `Fable.Elmish.React`, you also install `Fable.React` because it is a *transitive* dependency. This means now you have to install `react` and `react-dom` yourself because you installed `Fable.React`.
+
+This is something you just have to know and applies to every Fable binding you use. Wouldn't it be nice if you didn't have to do this manually, having a tool that does this for you correctly and automatically? Here comes Femto into play.
+
+### What is Femto
+
+Femto is a dotnet CLI tool that primarily does two things
+ - **Project dependency analysis**: going through all your project's dependencies, the direct and the transitive, extracting information about which npm dependencies are needed, which versions they should have and where they should be installed. Next this information is compared against what you already have installed to decide whether a package is missing or if a version of an installed npm package has an invalid version (falls outside the required version constraint specified by the Fable binding). Femto will log the commands that are required for full package resolution.
```

Comment:
> and where they should be installed

Isn't there only one place where they can be installed?

Reply:
dependencies vs devDependencies, I will mention that too